src/components/app/DocumentList/index.js

```diff
@@ -300,9 +304,10 @@ class DocumentListContainer extends Component {
       fetchLayout,
       updateRawModal,
       setModalDescription,
+      isModal,
     } = this.props;
 
-    fetchLayout(windowType, type, viewProfileId)
+    fetchLayout(windowType, type, viewProfileId, isModal ? viewId : null)
```

Comment:
Can we not have such logic and pass only a param name ?  There are way too many params anyway in here I understand that that cannot be changed now and pass only one single object but let's just pass  a variable in here with a meaningful name in here and it's fine.

Reply:
Yeah, I know. Will refactor this as I might be working near this code now.